packages/synthetix-main/contracts/storage/VaultEpoch.sol

```diff
@@ -3,47 +3,116 @@ pragma solidity ^0.8.0;
-        /// @dev tracks usd for each user
-        Distribution.Data usdDebtDist;
+        /**
+         * @dev Tracks consolidated debt for each user.
+         *
+         * Updated when users interact with the system, consolidating changes from the fluctuating incomingDebtDist,
+         * and directly when users mint or burn USD, or repay debt.
+         */
+        Distribution.Data consolidatedDebtDist;
     }
 
+    /**
+     * @dev Converts an account id to an actor id used in Distribution objects.
+     *
+     * TODO: Consider moving this into a Distribution helper, and use everywhere.
+     * Seeing multiple uses of bytes32(uint(uint | address)), in the code.
+     */
     function accountToActor(uint128 accountId) internal pure returns (bytes32) {
         return bytes32(uint(accountId));
     }
 
+    /**
+     * @dev Updates the value per share of the incoming debt distribution.
+     * Used for socialization during liquidations, and to bake in market changes.
+     *
+     * Called from:
+     * - LiquidationModule.liquidate
+     * - Pool.recalculateVaultCollateral (ticker)
+     */
     function distributeDebt(Data storage self, int debtChange) internal {
-        self.debtDist.distributeValue(debtChange);
+        self.incomingDebtDist.distributeValue(debtChange);
 
-        // total debt unfortunately needs to be cached here for liquidations
-        self.unclaimedDebt += int128(debtChange);
+        // Cache total debt here.
+        // Will roll over to individual users as they interact with the system.
+        self.unconsolidatedDebt += int128(debtChange);
     }
 
+    /**
+     * @dev Consolidates user debt as they interact with the system.
+     *
+     * Fluctuating debt is moved from incoming to consolidated debt.
+     *
+     * Called as a ticker from various parts of the system, usually whenever the
+     * real debt of a user needs to be known.
+     *
+     * TODO: Rename to consolidateAccountDebt?
+     */
     function updateAccountDebt(Data storage self, uint128 accountId) internal returns (int currentDebt) {
         bytes32 actorId = accountToActor(accountId);
 
-        currentDebt = self.usdDebtDist.getActorValue(actorId);
-        int newDebt = self.debtDist.accumulateActor(actorId);
+        currentDebt = self.consolidatedDebtDist.getActorValue(actorId);
+        int newDebt = self.incomingDebtDist.accumulateActor(actorId);
 
         currentDebt += newDebt;
 
-        self.usdDebtDist.updateActorValue(actorId, currentDebt);
-        self.unclaimedDebt -= int128(newDebt);
+        self.consolidatedDebtDist.updateActorValue(actorId, currentDebt);
+        self.unconsolidatedDebt -= int128(newDebt);
     }
 
+    /**
+     * @dev Updates a user's collateral value, and sets their exposure to debt
+     * according to the collateral they delegated and the leverage used.
+     *
+     * Called whenever a user's collateral changes.
+     *
+     * TODO: Rename to updateAccountCollateralAndDebtExposure?
```

Comment:
I like `setAccount` naming here because ultimately this is the function you want to call if you are changing an account's position within the vault. Maybe we can find a function name that is less wordy/more obvious to its importance?

Reply:
How about `updateAccountPosition`?

Reply:
:+1: